lib/runtime2/threepio/utils/LLMMonitoringService.ts

```diff
@@ -0,0 +1,147 @@
+import { JsonObject } from './jsonlProcessor';
+
+interface JsonSegmentTiming {
+  startTime: number;
+  endTime: number;
+  durationMs: number;
+  jsonContent: JsonObject;
+  rawLine: string;
+}
+
+// Interface for the data passed to addJsonOutputTiming by the client
+interface IncomingJsonSegmentData {
+  endTime: number;
+  jsonContent: JsonObject;
+  rawLine: string;
+}
+
+export interface MonitoredLLMCallData {
+  callId: string;
+  parentId?: string;
+  prompt?: string;
+  startTime: number;
+  endTime: number;
+  inputTokens?: number;
+  outputTokens?: number;
+  error?: string;
+  jsonOutputTimings: JsonSegmentTiming[];
+  childCalls?: MonitoredLLMCallData[];
+}
+
+export class LLMMonitoringService {
+  static instance: LLMMonitoringService;
```

Comment:
```suggestion
  static Instance: LLMMonitoringService = new LLMMonitoringService();
```
这样写了之后，就不需要单独的 getInstance 方法了。